zio-http/src/main/scala/zio/http/URL.scala

```diff
@@ -170,11 +171,11 @@ final case class URL(
   def queryParams(queryParams: QueryParams): URL =
     copy(queryParams = queryParams)
 
-  def queryParams(queryParams: Map[String, Chunk[String]]): URL =
+  def queryParams(queryParams: ListMap[String, Chunk[String]]): URL =
```

Comment:
I think this should just be `Map` not `ListMap`.
If the user don't care about the order, we don't need to force him.
```suggestion
  def queryParams(queryParams: Map[String, Chunk[String]]): URL =
     copy(queryParams = QueryParams(ListMap.from(queryParams)))
```

This will still keep the order if the `Map` is in fact a `ListMap`